charts/kube-prometheus-stack/Chart.lock

```diff
@@ -4,15 +4,15 @@ dependencies:
   version: 0.0.0
 - name: kube-state-metrics
   repository: https://prometheus-community.github.io/helm-charts
-  version: 5.14.0
+  version: 5.25.1
 - name: prometheus-node-exporter
   repository: https://prometheus-community.github.io/helm-charts
-  version: 4.23.2
+  version: 4.39.0
 - name: grafana
   repository: https://grafana.github.io/helm-charts
-  version: 6.60.6
+  version: 8.5.1
```

Comment:
This also upgrade Grafana to v8.5.1 https://github.com/grafana/helm-charts/releases/tag/grafana-8.5.1 
I am unable to tag Jayanth here somehow. Can you ask him to experiment with this version?